### PR TITLE
Update GHA to match org standards

### PR DESCRIPTION
**Pull request by NeonDaniel**
Update GHA to match other projects
Remove duplicated scripts

### CODE REVIEW

File: .github/workflows/publish_alpha.yml [hunk 1]
```diff
@@ -6,66 +6,43 @@ on:
     branches:
       - dev
     paths-ignore:
-      - 'ovos_vad_plugin_silero/version.py'
+      - 'ovos_plugin_manager/version.py'
```
Comment (mikejgray) [inline .github/workflows/publish_alpha.yml]: Looks like a reversion?

File: .github/workflows/publish_alpha.yml [hunk 2]
```diff
@@ -6,66 +6,43 @@ on:
...
+      major_var: VERSION_MAJOR
+      update_changelog: True
+      branch: dev
   build_and_publish:
     runs-on: ubuntu-latest
+    needs: update_version
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout Repository
+        uses: actions/checkout@v2
```
Comment (mikejgray) [inline .github/workflows/publish_alpha.yml]: Recommend v4

File: .github/workflows/publish_release.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  github_release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (mikejgray) [inline .github/workflows/publish_release.yml]: Recommend v4

**Review by mikejgray — CHANGES_REQUESTED** [review]

**Comment by JarbasAl** [comment]
using neon automations has caused issues in several repos, see the "hotfix" in several of them to skip making a github release so they could get published to pypi

idk whats wrong, only that release step is consistently failing in all OVOS repos that adopted it

**Comment by NeonDaniel** [comment]
> using neon automations has caused issues in several repos, see the "hotfix" in several of them to skip making a github release so they could get published to pypi
> 
> idk whats wrong, only that release step is consistently failing in all OVOS repos that adopted it

I've fixed this in a number of them; the issue has been a relative path used in `setup.py` which causes issues if you call `<path>/setup.py`

i.e.

https://github.com/OpenVoiceOS/OVOS-workshop/pull/125/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R5
https://github.com/OpenVoiceOS/ovos-PHAL-plugin-wallpaper-manager/pull/2/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7

**Review by mikejgray — APPROVED** [review]

**Comment by JarbasAl** [comment]
> > using neon automations has caused issues in several repos, see the "hotfix" in several of them to skip making a github release so they could get published to pypi
> > idk whats wrong, only that release step is consistently failing in all OVOS repos that adopted it
> 
> I've fixed this in a number of them; the issue has been a relative path used in `setup.py` which causes issues if you call `<path>/setup.py`
> 
> i.e.
> 
> [OpenVoiceOS/OVOS-workshop#125 (files)](https://github.com/OpenVoiceOS/OVOS-workshop/pull/125/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R5) [OpenVoiceOS/ovos-PHAL-plugin-wallpaper-manager#2 (files)](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-wallpaper-manager/pull/2/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7)

dont think that was it, since the first time the automation worked, but made a release with "V" only as name (no actual version number afterwards). the second time is when it would fail as it was trying to create release tag "V" and it already existed

but i dont know what the actual issue was, just noticed the pattern

**Pull request merged**